tests/test_from_pandoc.py

```diff
@@ -0,0 +1,30 @@
+from n2y.notion import Client
+from n2y.utils import id_from_share_link
+
+
+url = "a36972712dd24d808bb0107bf0945f41"
+token = "secret_aBbNlu5xsTtBNarAmuPYliS6pSCMpNaIJkcsz5eFuZn"
+client = Client(token, plugins=None)
+object_id = id_from_share_link(url)
+page = client.get_page_or_database(object_id)
```

Comment:
I like that you're extracting out code that needs to be run before every test. 

The problem with having these in the global scope like this is that the tests are no longer independent. For example, you're re-using the same `client` in each of your tests so if one test alters the state of your `client`, another test might be affected inadvertently by that. This can lead to strange or flaky test failures.

Instead, I'd recommend either wrapping all the non-literal variables (client, page) in a helper method or better yet, using pytest's fixtures: https://docs.pytest.org/en/6.2.x/fixture.html. They're designed for exactly this use case. 

Hope that makes sense.

Reply:
pytest fixtures are really cool; thanks for the tidbit. Definitely the perfect place to employ them